### ISSUE TITLE
Assume default value to created_at and updated_at attributes

### ISSUE THREAD

**Issue by ghost** [janjiss/rom_factory#4]
When creating a factory it should take on a default value to created_at and updated_at attributes.

Actual Result: Sequel::NotNullConstraintViolation: PG::NotNullViolation: ERROR:  null value in column "created_at" violates not-null constraint

Expected Result: set "created_at" column with current DateTime automatically.

**Comment by janjiss** [comment]
Hmm, thinking about DSL here. Maybe having something like:
```ruby
RomFactory::Builder.define do |b|
  b.factory(relation: :users, name: :user) do |f|
    f.timestamps
  end
end
```
What do you think?

**Comment by ghost** [comment]
@janjiss Very good, this way you might choose whether or not have timestamps

**Comment by janjiss** [comment]
@thiagopecanha Just published a version `0.3.1` that includes timestamps. Check out README for more information. Cheers
